### ISSUE TITLE
Lombok 1.18.2 is not compatible with auto compile on save feature of Apache Netbeans 9.0

### ISSUE THREAD

**Issue by privatejava** [rzwitserloot/lombok#1833]
Hi,
I have been working in Netbeans using lombok quiet a long time. I found that it is becoming unstable with the new Apache Netbeans 9.0 auto compile features when working in multi module maven projects. 

Project structure:
```
.
├── mod1
│   ├── pom.xml
│   └── src
│       ├── main
│       │   └── java
│       │       └── np
│       │           └── com
│       │               └── ngopal
│       │                   └── mod1
│       │                       └── User.java
│       └── test
│           └── java
├── mod2
│   ├── pom.xml
│   └── src
│       ├── main
│       │   └── java
│       │       └── np
│       │           └── com
│       │               └── ngopal
│       │                   └── mod2
│       └── test
│           └── java
│               └── HelloTest.java
└── pom.xml
```
_User.java_
```
@Data
public class User {
    private String msg;
    
    public void hello(){
        setMsg("Hello 8");
        System.out.println(msg);
    }
}
```
_HelloTest.java_
```
public class HelloTest {
    @Test
    public void test(){
        User u = new User();
        u.hello();
    }
}
```

So i have two modules where `mod2` depends upon `mod1` and I just wrote a `HelloTest.java` junit test for printing the `msg` from `User` class. 

**Lombok Version 1.16.10 and Apache Netbeans 9.0** _(Works good but the Netbeans editor shows red error regardless of successful compile)_ 
Step 1: I changed `setMsg("Hello 8")` -> `setMsg("Hello 9")` in _User.java_
Step 2: I ran Test Project in `mod2` It correctly displays `Hello 9`

**Lombok Version 1.16.10 and Apache Netbeans 9.0** _(Does not work at all but editor does not show any red error which is good )_
Step 1: I changed  `setMsg("Hello 9")` -> `setMsg("Hello 10")` in _User.java_
Step 2: I ran Test Project in `mod2` It displays `Hello 9` 

Currently it works only by re-compiling `mod1` manually every time I change some code in `mod1` module.

**Comment by rzwitserloot** [comment]
Sounds like a duplicate of #1770 – can you give the edge release a try? It contains the fix. https://projectlombok.org/download-edge

**Comment by privatejava** [comment]
I tried the latest build of lombok edge.
```
mvn install:install-file -DgroupId=org.projectlombok -DartifactId=lombok-edge -Dpackaging=jar -Dversion=1.18.3 -Dfile=lombok-edge.jar -DgeneratePom=true
```
Updated my dependencies and tried clean build which cleared and installed artifact new one with new lombok. But still Netbeans cannot pickup edits of `mod1` in `mod2` during the JUnit test . However during the Netbeans editor mode if I edit any files in `User.java` by adding any new method inside, these methods were quickly taken up in `HelloTest.java` which is good.

Thanks
Narayan
